### ISSUE TITLE
[BUG/ISSUE] Lightning flash Rate_save unit conversion issue in HEMCO hcox_lightnox_mod.F90

### ISSUE THREAD

**Issue by rgryan92** [geoschem/geos-chem#795]
In hcox_lightnox_mod.F90, line 447, the lightning flash rate term RATE_SAVE should be calculated by conversion from flashes/km2/s to flashes/km2/min. Lines 446-447 read

```
! Flashes per km2 per minute
RATE_SAVE   = RATE / 360d0
```

I think it should be divided by 60 to convert from /s to /min.#

The result does not transfer to the lightning flash rate used in lightning NOx calculations but does mean that the lightning flash rate output by HEMCO diagnostics is not correct.

@ltmurray @eamarais

**Comment by ltmurray** [comment]
This is correct, and a hold-over from when lightning was calculated online within the model, instead of prescribed as a meteorological field. I have created a pull request with a fix.

**Comment by ltmurray** [comment]
@yantosca can you move this ticket over to HEMCO?

**Comment by rgryan92** [comment]
Thanks @ltmurray
